pkg/rpc/server/server.go

```diff
@@ -105,6 +143,9 @@ const (
 
 	// Maximum number of elements for get*transfers requests.
 	maxTransfersLimit = 1000
+
+	// defaultSessionPoolSize is the number of concurrently running iterator sessions.
+	defaultSessionPoolSize = 1000
```

Comment:
20? 1000 seems like an overkill.

Reply:
Still 1000 here and 200 in the doc. And why not 20?

Reply:
Ugh, sorry, fixed.